Install netcat and jq in the common stage

fortum-vaanavil opened the pull request:
The wait-for.sh script requires netcat and currently the `client-standalone` image fails if trying to use the `wait-for.sh` script, since netcat happens to not be installed in that specific image.

## Changes proposed

Move the installation of `netcat` and `jq` to the common stage, since `netcat` is required by the `wait-for.sh` script

<!-- List all the proposed changes in your PR -->

<!-- Mark all the applicable boxes. To mark the box as done follow the following conventions -->
<!--
[x] - Correct; marked as done
[X] - Correct; marked as done

[ ] - Not correct; marked as **not** done
-->

## Check List (Check all the applicable boxes) <!-- Follow the above conventions to check the box -->

- [x] I sign off on contributing this submission to open-source
- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

asafc commented:
Hi @fortum-vaanavil, thanks for the contribution! :)
It might take us a couple days to review due to workload (it's on @roekatz 's queue, no worries :))

roekatz reviewed (changes requested):
Please remove duplications of installing `nectat` and `jq` in the `client` & `server` stages

fortum-vaanavil commented:
> Hi @fortum-vaanavil , I'm not sure what you're trying to accomplish here. Both server & client images already had both `nectat` & `jq`.
> 
> If you're trying to organize the `Dockerfile` by moving common things to the `common` stage (which is a welcome change), go ahead and remove duplications because other stages in the file still install those packages again :)

Yeah sorry, I realized I completely failed to document the actual issue and also missed the duplicate installations. I've now updated the PR description and removed the duplicates.
